modules/nf-core/gatk4/splitncigarreads/environment.yml

```diff
@@ -4,4 +4,4 @@ channels:
   - bioconda
   - defaults
 dependencies:
-  - bioconda::gatk4=4.4.0.0
+  - bioconda::gatk4=4.5.0.0
```

Comment:
updating version too?

Reply:
yeah, so I don't have to re-run the tests when I merge it into my other big PR